sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -28,13 +28,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Remove="Shared\*.cs"/>
-    <Compile Include="Shared\Argument.cs"/>
-    <Compile Include="Shared\EventSourceEventFormatting.cs"/>
-    <Compile Include="Shared\HashCodeBuilder.cs"/>
-    <Compile Include="Shared\NullableAttributes.cs"/>
-    <Compile Include="Shared\ContentTypeUtilities.cs"/>
-    <Compile Include="Shared\ClientDiagnostics.cs"/>
-    <Compile Include="Shared\DiagnosticScope.cs"/>
+    <Compile Remove="Shared\*.cs" />
```

Comment:
Do we need this change to Azure.Core?

Reply:
No. And I have no idea why did it happen :)